src/wallet/asyncrpcoperation_common.cpp

```diff
@@ -117,7 +117,13 @@ void ThrowInputSelectionError(
             }
         },
         [&](const InvalidFundsError& err) {
-            bool isFromUa = std::holds_alternative<libzcash::UnifiedAddress>(selector.GetPattern());
+            bool suggestLinkingAddresses = std::visit(match {
+                    [](const libzcash::UnifiedAddress&) { return true; },
```

Comment:
Should this also be
```suggestion
                    [](const libzcash::UnifiedAddress&) {
                        return !strategy.AllowLinkingAccountAddresses();
                    },
```
(to avoid suggesting `AllowLinkingAccountAddresses` when the policy already allows it)?
Or can that case not occur?

Reply:
Right – it shouldn’t be able to occur. If we allow linking, then it should have been converted to an account pattern. But … that only happens to be the case. Your suggestion makes it more robust, so it’s worth doing. But perhaps like
```
bool suggestLinkingAddresses = !strategy.AllowLinkingAccountAddresses() && std::visit(match {
    [](const libzcash::UnifiedAddress&) { return true; },
    [](const AccountZTXOPattern&) { return true; },
    ...
```
(plus using `examine` instead of `std::visit`)